CHANGELOG.md

```diff
@@ -4,7 +4,9 @@
 
 ### Changed
 
-- NuvlaEdge details - Fixes opening Dropdown menu on key presses inside Update NuvlaEdge form Env Vars bugfix #908
+- NuvlaEdge details - Fixes opening Dropdown menu on key presses inside Update NuvlaEdge form Env Vars bugfix #908<<<<<<< issue_901
```

Comment:
@MTrost what is meant by `#908<<<<<<< issue_901`? Or it's a typo?

Reply:
Bad merge conflict solution, fixed now.